src/pages/billing/List.js

```diff
@@ -51,6 +51,32 @@ const billingProjectNameValidator = existing => ({
   }
 })
 
+const noBillingMessage = div({ style: { fontSize: 20, margin: '2rem' } }, [
+  div([
+    'To get started, click', icon('plus-circle', {
+      size: 21, style: { color: colors.accent(), margin: '0 0.5rem' }
```

Comment:
Having the icon here makes it look clickable, which could be confusing. Perhaps we should just describe it with text, e.g. 'Click the plus button'